classes/press/NewReleaseDAO.inc.php

```diff
@@ -66,7 +66,10 @@ function getMonographsByAssoc($assocType, $assocId) {
 		$publishedMonographDao = DAORegistry::getDAO('PublishedMonographDAO');
 		while (!$result->EOF) {
 			list($monographId) = $result->fields;
-			$returner[] = $publishedMonographDao->getById($monographId);
```

Comment:
This change could be made more compactly and efficiently by moving it directly into the query above. I'll merge this (barring anything major) and make that change.